openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts

```diff
@@ -222,23 +221,20 @@ export const deleteEntity = async (
   entityType: string,
   entityId: string,
   isRecursive: boolean,
-  isSoftDelete = false
+  isHardDelete = true
 ) => {
   let path = '';
 
-  if (isSoftDelete) {
-    path = getURLWithQueryFields(`/${entityType}/${entityId}`);
-  } else {
-    const searchParams = new URLSearchParams({ hardDelete: `true` });
-    if (!isUndefined(isRecursive)) {
-      searchParams.set('recursive', `${isRecursive}`);
-    }
-    path = getURLWithQueryFields(
-      `/${entityType}/${entityId}`,
-      '',
-      `${searchParams.toString()}`
-    );
-  }
+  const searchParams = new URLSearchParams({
+    hardDelete: `${isHardDelete}`,
+    recursive: `${isRecursive}`,
+  });
+
+  path = getURLWithQueryFields(
+    `/${entityType}/${entityId}`,
+    '',
+    `${searchParams.toString()}`
+  );
 
   return APIClient.delete(path);
 };
```

Comment:
can we use params directly, as we have done at other places? let me know if you need any help!

Reply:
Done 👍